projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -2708,6 +2708,13 @@ public static function get_updateable_data_list( $selector = '' ) {
 				'validate_callback' => __CLASS__ . '::validate_verification_service',
 				'jp_group'          => 'verification-tools',
 			),
+			'facebook'				=> array(
```

Comment:
You'll need to use spaces for indenting here:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation

> if you have a block of code that would be more readable if things are aligned, use spaces:

Reply:
Oops, thank you! Should be fixed now.